### PR TITLE
fix: disable auto login when refresh page

### PR DESCRIPTION
**Pull request by keeplearning-near**

### CODE REVIEW

**Comment by zhubajie8881** [comment]
test passed

File: src/libs/web3-data-provider/provider.tsx [hunk 1]
```diff
@@ -363,17 +364,9 @@ export function Web3Provider({
...
-        // see the source of the `isSafeApp()` function
-        // if (window?.parent === window) {
-        //   return false;
-        // }
-        // in pret app, we only need browser support by self now
-        // no need to connect a gnosis-safe signer wallet
-        storedProviderName = 'browser';
       }
       if (storedProviderName) {
+        console.log('storedProviderName', storedProviderName);
```
Comment (linguists) [inline src/libs/web3-data-provider/provider.tsx:369]: Can we remove the log?

**Review by linguists — CHANGES_REQUESTED** [review]

File: src/libs/web3-data-provider/provider.tsx (hunk 1 above)
Comment (SteveYuOWO) [inline src/libs/web3-data-provider/provider.tsx:369]:
```suggestion
```

Comment (SteveYuOWO) [reply src/libs/web3-data-provider/provider.tsx:369]: yes

**Review by SteveYuOWO — APPROVED** [review]

**Review by SteveYuOWO — APPROVED** [review]

File: src/libs/web3-data-provider/provider.tsx (hunk 1 above)
Comment (keeplearning-near) [inline src/libs/web3-data-provider/provider.tsx:369]:
```suggestion
```

**Review by linguists — APPROVED** [review]

**Pull request merged**